lib/subscription/subscription-store.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference path='../../typings/tsd.d.ts' />
+
+import _ = require('lodash');
+import Subscription = require('./subscription');
+
+export = SubscriptionStore;
+
+class SubscriptionStore<T extends Subscription> {
```

Comment:
This is similar in functionality to `Object`, I would almost argue that we could forgo this class as well.  Thoughts?